### PR TITLE
Add Nines by Ungodly Design

### PR DESCRIPTION
**Pull request by luis-godinez**
## Description

Add qmk support for 3x3 macropad with dual rotary encoders.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/ungodly/nines/rules.mk [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+ENCODER_ENABLE = yes        # Rotary encoder support
```
Comment (fauxpark) [inline keyboards/ungodly/nines/rules.mk:22]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
ENCODER_ENABLE = yes        # Rotary encoder support
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ungodly/nines/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+[2] = LAYOUT_ortho_3x3(
+    _______, _______, _______,
+    _______, _______, _______,
+    _______, _______, _______
+),
+
+[3] = LAYOUT_ortho_3x3(
+    _______, _______, _______,
+    _______, _______, _______,
+    _______, _______, _______
```
Comment (Erovia) [inline keyboards/ungodly/nines/keymaps/default/keymap.c:41]:
```suggestion
```

File: keyboards/ungodly/nines/readme.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Made by Ungodly Design for Qlavier.
+
+* Keyboard Maintainer: [Luis Godinez](https://github.com/luis-Godinez)
+* Hardware Supported: The Nines PCB
+* Hardware Availability: [Qlavier](https://www.qlavier.com/shop/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ungodly/nines:default
+
```
Comment (Erovia) [inline keyboards/ungodly/nines/readme.md:15]: Can you add "clear instructions on how to reset the board into bootloader mode" as per the [PR Checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

**Review by Erovia — CHANGES_REQUESTED** [review]

File: keyboards/ungodly/nines/nines.c [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ #include "nines.h"
+
+void encoder_update_kb(uint8_t index, bool clockwise) {
```
Comment (zvecr) [inline keyboards/ungodly/nines/nines.c:18]:
```suggestion
__attribute__((weak)) void encoder_update_user(uint8_t index, bool clockwise) {
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by Erovia — APPROVED** [review]

**Pull request merged**